test/edu-benefits/pages/applicantService.unit.spec.jsx

```diff
@@ -4,10 +4,10 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 import ReactTestUtils from 'react-addons-test-utils';
 
-import { DefinitionTester, submitForm } from '../../../util/schemaform-utils.jsx';
-import formConfig from '../../../../src/js/edu-benefits/5490/config/form';
+import { DefinitionTester, submitForm } from '../../util/schemaform-utils.jsx';
+import formConfig from '../../../src/js/edu-benefits/5490/config/form';
 
-describe('Edu 5490 applicantService', () => {
+describe('Edu 5490 (default) applicantService', () => {
```

Comment:
Do I need to do anything more than this here? The coverage is the same for both the 5490 and 5495.